src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.scss

```diff
@@ -1,67 +1,52 @@
-.alerts {
-  height: 17rem;
+.details {
+  dt {
+    font-size: larger;
+    margin-bottom: 0.3rem;
+  }
 
-  div {
-    padding-top: 0;
+  dd {
+    font-size: larger;
+    margin-bottom: 0.8rem;
   }
 }
 
-div {
-  padding-top: 20px;
+.status {
+  .viewAlert {
+    position: absolute;
+    right: 25px;
+    top: 15px;
```

Comment:
If instead of pixels we use a relative dimension (like `rem`) then it'll be easier to achieve screen size responsiveness.

Reply:
ack